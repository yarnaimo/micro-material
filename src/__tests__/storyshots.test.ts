import initStoryshots from '@storybook/addon-storyshots'
import serializer from 'jest-emotion'

initStoryshots({ snapshotSerializers: [serializer] })
