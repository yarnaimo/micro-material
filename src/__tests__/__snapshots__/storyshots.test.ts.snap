// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button Basic 1`] = `
.emotion-1 * {
  box-sizing: border-box;
}

.emotion-1 button,
.emotion-1 input,
.emotion-1 optgroup,
.emotion-1 select,
.emotion-1 textarea {
  font-family: inherit;
}

.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  margin: 0;
  padding: 0;
  border: none;
  outline: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 600;
  background: #FCB645;
  color: #FFFFFF;
  border-radius: 6px;
  padding-right: 11px;
  padding-left: 11px;
  height: 32px;
  min-width: 72px;
  font-size: 12px;
}

.emotion-0::before,
.emotion-0::after {
  background-color: #FFFFFF;
}

<div
  className="emotion-1"
>
  <button
    className="mdc-ripple-surface emotion-0"
    color="#FCB645"
    height={32}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    style={Object {}}
  >
    ラベル
  </button>
</div>
`;

exports[`Storyshots Button Icon 1`] = `
.emotion-2 * {
  box-sizing: border-box;
}

.emotion-2 button,
.emotion-2 input,
.emotion-2 optgroup,
.emotion-2 select,
.emotion-2 textarea {
  font-family: inherit;
}

.emotion-1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  margin: 0;
  padding: 0;
  border: none;
  outline: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-weight: 600;
  background: #FCB645;
  color: #FFFFFF;
  border-radius: 50%;
  width: 40px;
  height: 40px;
}

.emotion-1::before,
.emotion-1::after {
  background-color: #FFFFFF;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  width: 18px;
  height: 18px;
}

<div
  className="emotion-2"
>
  <button
    className="mdc-ripple-surface emotion-1"
    color="#FCB645"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    size={40}
    style={Object {}}
  >
    <div
      className="emotion-0"
      size={18}
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"100%\\" height=\\"100%\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" stroke=\\"currentColor\\" stroke-width=\\"3\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" class=\\"feather feather-heart\\"><path d=\\"M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z\\"></path></svg>",
          }
        }
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </button>
</div>
`;
